### ISSUE TITLE
the input channel order

### ISSUE THREAD

**Issue by mangdian** [gulvarol/ltc#2]
I am not sure about the input channel order.the input size is 3,16,112*112(three channels),but channel order should be RGB?or BGR?

**Comment by gulvarol** [comment]
I used RGB order for training from scratch, but I'm sorry I couldn't find time to complete the TO-DO's on the readme yet. The models I will release will be RGB order. If you are training from scratch, it doesn't matter. [Here](https://github.com/gulvarol/ltc/blob/master/donkey.lua#L88), I put an option for using pre-trained models with BGR order. C3D model is converted from caffe and there, it should be BGR. By default `opt.bgr` is false. See [here](https://github.com/gulvarol/ltc/blob/master/opts.lua#L44).

**Comment by mangdian** [comment]
https://github.com/gulvarol/ltc/releases/
Thank you for responding.Is this the model you released?So what's the channel order?I am a little confused..........

**Comment by gulvarol** [comment]
Yes I released this upon request after the issue #1. The order for this model should be BGR. I am planning to release instructions how to use it etc. once I have time.

**Comment by mangdian** [comment]
well,I input some data to the network,and find that if the channel order is set to RGB rather than BGR,the prediction accuracy  does not change,but the probability do increase.If the correct channel order is RGB,this is strange.
Besides,now I want to finetune this model based on torch on ActivityNet and Thumos,would this be 
feasible？

**Comment by gulvarol** [comment]
Please be more precise.

What do you mean by 'prediction accuracy does not change'? Do you train an SVM on top of last layer? In that case it is normal that you get similar prediction accuracy if you use RGB or BGR.

What do you mean by 'probability do increase'? Do you mean the output of last layer is higher when you input BGR? As I said in my previous reply, the correct channel order for this particular model should be BGR.

What do you mean by 'feasible'? In terms of coding, yes it is trivial. But I cannot help you further with this.

**Comment by mangdian** [comment]
I mean,the output of last layer is higher when I input RGB,which makes me confused,because you said that the correct order should be BGR

**Comment by mangdian** [comment]
I found that the released C3D model is with dropout.how can I disable dropout of fc layers?

**Comment by gulvarol** [comment]
Yes because C3D architecture has dropout layers. You can remove them if you want. I am closing this issue as this question is unrelated to the title.
